test/test_helper.rb

```diff
@@ -35,3 +38,9 @@
 
 require 'minitest/autorun'
 require 'rack/test'
+
+module FakeRedis
+  def teardown
+    RouterWrapper.config[:redis_count].flushall
+  end
+end
```

Comment:
empty line missing at the end: your editor is not correctly configured!

Reply:
done.